src/main.rs

```diff
@@ -28,6 +29,12 @@ fn main() -> ExitCode {
 fn run() -> Result<(), Box<dyn std::error::Error>> {
     tree::ui::init();
```

Comment:
Would you mind moving this piece of code to below `if let Some(shell) = ctx.completions { ... }`? Just because there's no need to initialize all the UI code when setting up completions.

Reply:
Done